all: change to use inclusive language

mhilton opened the pull request:
Update the codebase to use more inclusive language. This introduces a
non backwards-compatible change in the configuration file, this is in
an undocumented setting that is not know to be used outside of canonical
at this point.

jujugui commented:
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candid/127/

mhilton commented:
:shipit:

jujugui commented:
Status: merge request accepted. Url: http://ci-gce.theblues.io:8080/job/candid-merge

mhilton committed:
all: change to use inclusive language

Update the codebase to use more inclusive language. This introduces a
non backwards-compatible change in the configuration file, this is in
an undocumented setting that is not know to be used outside of canonical
at this point.

jcsackett reviewed (approved):
Thanks @mhilton!
